EXT_LaserEnvelope.md

```diff
@@ -7,14 +7,14 @@ openPMD extension name: `LaserEnvelope`
 Introduction
 ------------
 
-This extension is specifically designed for the modeling laser beam propagation.
+This extension is specifically designed for modeling the laser beam propagation.
```

Comment:
Minor idea:
```suggestion
This extension is specifically designed for modeling the propagation of laser beams and pulses.
```

Reply:
Yeah, I think that works well, we should be able to support monochromatic beams.